---
layout: default
---
<body>

<table style="width:100%">
<tr>
<td>
<p><b>Contact:</b></p>
<p>Bioinformatics group</br>
Department of Plant Sciences</br>
Wageningen University & Research</br>
Wageningen, Netherlands</p>
<p><b>E-mail:</b> anne . kupczok @ wur . nl</p>
</td>
<td>
<p><img src="KEC_small.png" alt="Anne" /></p>
</td>
</table>

<h2>Research focus</h2>

<p><b>Evolutionary rate estimation</b><p>

<p>Estimation of absolute substitution, recombination and gene turnover rates from time series data sets.</p>

<p><b>Phage pangenome evolution</b><p>

<p>Contribution of different evolutionary processes to phage genome evolution.</p>

<p><b>Evolutionary metagenomics</b><p>

<p>Inference of microbial evolutionary processes from genetic heterogeneity present in metagenomes.</p>

<hr>

<h2>Main ongoing projects</h2>

<p>Population genetics of <i>Bathymodiolus</i> symbionts (with <a href="https://www.mpi-bremen.de/en/Nicole-Dubilier.html">Nicole Dubilier</a>)</p>

<p>The casposon of <i>Methanosarcina mazei</i> Gö1 - function and evolution (with <a href="http://www.mikrobio.uni-kiel.de/de/ag-schmitz-streit">Ruth Schmitz-Streit</a>, Finn Ole Petersen, project in priority programme <a href="https://www.uni-ulm.de/nawi/crispr-spp2141/">Much more than defense: The many faces and facets of CRISPR-Cas</a>)</p>

<p>Models for pangenome evolution (with <a href="http://baumdickerlab.de/">Franz Baumdicker</a>)</p>

<p>Plasmid evolution (with <a href="https://www.mikrobio.uni-kiel.de/de/ag-dagan">Tal Dagan</a>, <a href="http://web.evolbio.mpg.de/~uecker/">Hildegard Uecker</a>, Mario Santer)</p>

<p>Evolution of lambda phages (with <a href="https://usys.ethz.ch/en/people/profile.MjUyNjM3.TGlzdC82MzcsMzIwMTk3MjIy.html">Carolin Wendling</a>)</p>

<hr>

<h2>Short CV</h2>

<h3>2020 - current</h3>

<p>Postdoctoral researcher in the group of <a href="https://www.mpi-bremen.de/en/Nicole-Dubilier.html">Nicole Dubilier</a>.</p>

<h3>2020</h3>

<p>Postdoctoral researcher in the group of <a href="https://www.mpi-bremen.de/en/Nicole-Dubilier.html">Nicole Dubilier</a>.</p>

<h3>2013 - 2020</h3>

<p>Postdoctoral researcher in the group of <a href="https://www.mikrobio.uni-kiel.de/de/ag-dagan">Tal Dagan</a>.</p>

<p>At Kiel University I started as a bioinformatician within the Excellence cluster <a href="https://www.futureocean.org/en/index.php">"The future ocean"</a>. Since 2015 my position was funded by the state of Schleswig-Holstein for maintaining the <a href="http://www.bioinf.uni-kiel.de/en">bioinformatics network</a> in the Kiel area.</p>

<h4>Main Publications</h4>
<p> Godfroid M, Dagan T, Merker M, Kohl TA, Diel R, Maurer FP, Niemann S, <b>Kupczok A</b>. 2020. Insertion and deletion evolution reflects antibiotics selection pressure in a Mycobacterium tuberculosis outbreak. bioRxiv:2020.01.28.922765.</p>

<p>Romero Picazo D, Dagan T, Ansorge R, Petersen JM, Dubilier N, <b>Kupczok A</b>. 2019. Horizontally transmitted symbiont populations in deep-sea mussels are genetically isolated. <i>ISME J.</i> 13:2954–2968.</p>

<p>Ilhan J, <b>Kupczok A</b>, Woehle C, Wein T, Hülter NF, Rosenstiel P, Landan G, Mizrahi I, Dagan T. 2019. Segregational drift and the interplay between plasmid copy number and evolvability. <i>Mol Biol Evol.</i> 36(3):472-486.</p>

<p>Godfroid M, Dagan T, <b>Kupczok A</b>. 2018. Recombination signal in Mycobacterium tuberculosis stems from reference-guided assemblies and alignment artefacts. <i>Genome Biol Evol.</i> 10(8):1920–1926.</p>

<p><b>Kupczok A</b>, Neve H, Huang KD, Hoeppner MD, Heller KJ, Franz CMAP, Dagan T. 2018. Rates of mutation and recombination in Siphoviridae phage genome evolution over three decades. <i>Mol Biol Evol.</i> 35(5):1147–1159.</p>

<p><b>Kupczok A</b>, Landan G, Dagan T. 2015. The contribution of genetic recombination to CRISPR array evolution. <i>Genome Biol Evol.</i> 7(7):1925–1939.</p>

<h3>2010 - 2013</h3>

<p>Postdoctoral researcher at the <a href="https://ist.ac.at/de/home/">Institute of Science and Technology, Austria (IST Austria)</a> in the group of Jonathan Bollback.</p>

<p>In Jon's group I learned to appreciate the microbial world for the study of evolutionary processes. For example, we used phylogenetic methods to study <a href="https://www.ncbi.nlm.nih.gov/pubmed/23442002">the evolution of the CRISPR/Cas immune system</a>.</p>

<h4>Main Publications</h4>

<p><b>Kupczok A</b>, Bollback JP. 2014. Motif depletion in bacteriophages infecting hosts with CRISPR systems. <i>BMC Genomics</i> 15:663.</p>

<p><b>Kupczok A</b>, Bollback JP. 2013. Probabilistic models for CRISPR spacer content evolution. <i>BMC Evol Biol.</i> 13:54.</p>

<h3>2006 - 2010</h3>

<p>PhD student at the <a href="https://www.maxperutzlabs.ac.at/">Max Perutz Labs</a>, Vienna, working in the Center for Integrative Bioinformatics Vienna (<a href="http://www.cibiv.at/">CIBIV</a>) headed by Arndt von Haeseler.</p>

<p>My PhD project focused on phylogenetics, in particular on method development for supertrees and tree distances. All the details can be found <a href="http://othes.univie.ac.at/9865/">here</a>.</p>

<h4>Main Publications</h4>

<p><b>Kupczok A</b>. 2011. Split-based computation of majority-rule supertrees. <i>BMC Evol. Biol.</i> 11:205.</p>

<p><b>Kupczok A</b>. 2011. Consequences of different null models on the tree shape bias of supertree methods. <i>Syst Biol.</i> 60(2):218–225.</p>

<p><b>Kupczok A</b>, Schmidt HA, von Haeseler A. 2010. Accuracy of phylogeny reconstruction methods combining overlapping gene data sets. <i>Algorithms Mol Biol.</i> 5:37.</p>

<p><b>Kupczok A</b>, von Haeseler A, Klaere S. 2008. An Exact Algorithm for the Geodesic Distance between Phylogenetic Trees. <i>J Comput Biol.</i> 15(6):577–591.</p>

<h4>Software</h4>

<p><a href="http://www.cibiv.at/software/geometree/">GeoMeTree</a> - Geodesic Metric on Trees (another branch can be found <a href="https://github.com/kloetzl/geometree">here</a>).</p>

<p><a href="http://www.cibiv.at/software/plumist/">PluMiST</a> - Plus and Minus SuperTrees.</p>

<h3>2003 - 2005</h3>

<p>Research assistant at the Friedrich Schiller University, Jena, working in the <a href="http://www.biosys.uni-jena.de/">Biosystems Analysis Group</a> headed by Peter Dittrich.</p>

<p>Peter gave me the freedom to have a research project already as an undergraduate.</p>

<h4>Main Publication</h4>

<p><b>Kupczok A</b>, Dittrich P. 2006. Determinants of simulated RNA evolution. <i>J. Theor. Biol.</i> 238:726–735.</p>

<h3>Education</h3>

<p>2020 - Habilitation (German post-doctoral lecturing qualification), Kiel University, Germany.</p>

<p>2010 - PhD in Molecular Biology, University of Vienna, Austria.</p>

<p>2006 - Diploma in Bioinformatics, Friedrich-Schiller-University Jena, Germany.</p>

<hr>

<h2>More Information</h2>

<p>Complete list of publications in <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Anne+Kupczok">pubmed</a>.</p>

<p>Profile on <a href="https://scholar.google.de/citations?user=XEdO4FwAAAAJ&hl=en">Google Scholar</a>.</p>

<p>Profile on <a href="https://orcid.org/0000-0001-5237-1899">ORCID</a>.</p>

<hr>

<a class="twitter-timeline" data-height="400" href="https://twitter.com/annecmg?ref_src=twsrc%5Etfw">Tweets by annecmg</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
</body>
