---
layout: default
---
<body>

<p><img src="KEC_small.png" alt="Anne" /></p>

<h2>Research interests</h2>

<p>Evolutionary metagenomics</p>

<p>Bacteriophage genome evolution</p>

<p>Evolutionary rate estimation</p>

<p>Evolution of the microbial immune system CRISPR/Cas

<hr>

<h2>Short CV</h2>

<p><b>2013-current</b></p>

<p>PostDoc in the group of <a href="https://www.mikrobio.uni-kiel.de/de/ag-dagan">Tal Dagan</a>.</p>

<p>At Kiel University I started as a bioinformatician for the Excellence cluster <a href="https://www.futureocean.org/en/index.php">"The future ocean"</a>. Since 2015 my position is funded by the state of Schleswig-Holstein for maintaining the <a href="http://www.bioinf.uni-kiel.de/en">bioinformatics network</a> in the Kiel area.</p>

<p><b>2010-2013</b></p>

<p>PostDoc at the <a href="https://ist.ac.at/de/home/">Institute of Science and Technology, Austria (IST Austria)</a> in the group of Jonathan Bollback.</p>

<p>In Jon's group I learned to appreciate the microbial world for the study of evolutionary processes. For example, we used phylogenetic methods to study <a href="https://www.ncbi.nlm.nih.gov/pubmed/23442002">the evolution of the CRISPR/Cas immune system</a>.</p>

<p><b>2006 - 2010</b></p>

<p>PhD student at the <a href="https://www.maxperutzlabs.ac.at/">MaxPerutz Labs</a>, Vienna, working in the Center for Integrative Bioinformatics Vienna (<a href="http://www.cibiv.at/">CIBIV</a>) headed by Arndt von Haeseler.</p>

<p>My PhD project focused on phylogenetics, in particular on method development for supertrees and tree distances. All the details can be found <a href="http://othes.univie.ac.at/9865/">here</a>.</p>

<p><b>2003-2005</b></p>

<p>Student assistant at the Friedrich Schiller University, Jena, working in the <a href="http://www.biosys.uni-jena.de/">Biosystems Analysis Group</a> headed by Peter Dittrich.</p>

<p>Peter gave me the freedom to have a research project already as an undergraduate. With an <a href="https://www.ncbi.nlm.nih.gov/pubmed/16098538">outcome</a>.</p>

<p><b>2001-2006</b></p>

<p>Diploma in Bioinformatics, Friedrich-Schiller-University Jena, Germany.</p>

<hr>

<h2>Main ongoing projects</h2>

<p>The life cycle of the deep-sea Bathymodiolus metaorganism: Population genetics of Bathymodiolus symbionts (collaboration with Devani Romero Picazo, Tal Dagan, <a href="https://www.mpi-bremen.de/en/Nicole-Dubilier.html">Nicole Dubilier</a>, project in <a href="https://www.metaorganism-research.com/">CRC 1182</a>)</p>

<p>The horizontal component of <i>Mycobacterium tuberculosis</i> genome evolution (collaboration with Maxime Godfroid, Tal Dagan, <a href="https://fz-borstel.de/index.php/de/sitemap/programmbereich-infektionen/molekulare-und-experimentelle-mykobakteriologie-prof-dr-stefan-niemann/mitarbeiter-innen">Stefan Niemann</a>, project in Leibniz science campus <a href="http://evolung.fz-borstel.de/">EvoLUNG</a>)</p>

<p>The casposon of <i>Methanosarcina mazei</i> Gö1 - function and evolution (collaboration with <a href="http://www.mikrobio.uni-kiel.de/de/ag-schmitz-streit">Ruth Schmitz-Streit</a>, project in priority programme <a href="https://www.uni-ulm.de/nawi/crispr-spp2141/">Much more than defense: The many faces and facets of CRISPR-Cas</a>)</p>

<p>...</p>

<hr>

<h2>Main publications</h2>

<p>Romero Picazo D, Dagan T, Ansorge R, Petersen JM, Dubilier N, <b>Kupczok A</b>. 2019. Horizontally transmitted symbiont populations in deep-sea mussels are genetically isolated. <i>ISME J.</i></p>

<p>Ilhan J, <b>Kupczok A</b>, Woehle C, Wein T, Hülter NF, Rosenstiel P, Landan G, Mizrahi I, Dagan T. 2019. Segregational drift and the interplay between plasmid copy number and evolvability. <i>Mol Biol Evol.</i> 36(3):472-486.</p>

<p>Godfroid M, Dagan T, <b>Kupczok A</b>. 2018. Recombination signal in Mycobacterium tuberculosis stems from reference-guided assemblies and alignment artefacts. <i>Genome Biol Evol.</i> 10(8):1920–1926.</p>

<p><b>Kupczok A</b>, Neve H, Huang KD, Hoeppner MD, Heller KJ, Franz CMAP, Dagan T. 2018. Rates of mutation and recombination in Siphoviridae phage genome evolution over three decades. <i>Mol Biol Evol.</i> 35(5):1147–1159.</p>

<p><b>Kupczok A</b>, Landan G, Dagan T. 2015. The contribution of genetic recombination to CRISPR array evolution. <i>Genome Biol Evol.</i> 7(7):1925–1939.</p>

<p><b>Kupczok A</b>, Bollback JP. 2014. Motif depletion in bacteriophages infecting hosts with CRISPR systems. <i>BMC Genomics</i> 15:663.</p>

<p><b>Kupczok A</b>, Bollback JP. 2013. Probabilistic models for CRISPR spacer content evolution. <i>BMC Evol Biol.</i> 13:54.</p>

<p><b>Kupczok A</b>. 2011. Consequences of different null models on the tree shape bias of supertree methods. <i>Syst Biol.</i> 60(2):218–225.</p>

<p><b>Kupczok A</b>, Schmidt HA, von Haeseler A. 2010. Accuracy of phylogeny reconstruction methods combining overlapping gene data sets. <i>Algorithms Mol Biol.</i> 5:37.</p>

<p><b>Kupczok A</b>, von Haeseler A, Klaere S. 2008. An Exact Algorithm for the Geodesic Distance between Phylogenetic Trees. <i>J Comput Biol.</i> 15(6):577–591.</p>

<p>Complete list of publications in <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Anne+Kupczok">pubmed</a>.</p>

<hr>

<a class="twitter-timeline" data-height="400" href="https://twitter.com/annecmg?ref_src=twsrc%5Etfw">Tweets by annecmg</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
</body>
